engine/consensus/matching/engine_test.go

```diff
@@ -591,7 +578,7 @@ func (ms *MatchingSuite) TestRequestPendingReceipts() {
 	}
 	ms.SealsPL.On("All").Return([]*flow.IncorporatedResultSeal{}).Maybe()
 
-	_, err := ms.matching.requestPendingReceipts()
+	_, _, err := ms.matching.requestPendingReceipts()
 	ms.Require().NoError(err, "should request results for pending blocks")
 	ms.requester.AssertExpectations(ms.T()) // asserts that requester.EntityByID(<blockID>, filter.Any) was called
 }
```

Comment:
Need a test where we try to add receipt with unknown previous receipt and check if it was added into mempool

Reply:
will add in 387